spec.bs

```diff
@@ -1436,6 +1451,25 @@ trust that the merchant showed the user the correct amount in their checkout
 flow (and any fraud discoveries are post-payment, when the user checks their
 account statement).
 
+## Lack of user activation requirement ## {#sctn-security-user-activation-requirement}
+
+If the user agent decides to skip the user activation requirement as outlined in
+[[#sctn-modify-user-activation-requirement]], some additional security mitigations
+should be considered. Removing this requirement increases the risk of spam and
+click-jacking attacks, by allowing a Secure Payment Confirmation flow to be
+initiated without the user's intent.
+
+In order to mitigate spam, the user agent may decide to enforce a user
+activation requirement after some threshold, for example after the user has
+already been shown a Secure Payment Confirmation flow without a user activation
+on the current page. Similarly, in order to mitigate click-jacking attacks, the
```

Comment:
```suggestion
on the current page. In order to mitigate click-jacking attacks, the
```

I don't think they that similar? These feel like two distinct mitigations for two distinct problems :)